index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Raj Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <title>Raj Portfolio</title>
+</head>
+ <body>
+  <main class="rj">
+   <header>
+      <a href="#"class="logo">My Logo</a>
+      <nav class="nav">
+          <div class="button"><a href="#"><img src="image/Union.png" alt="button"></a></div>
+      </nav>
+  </header>
+  <section>
+    <h1>Hey There. I'm Vivek Raj glad to see you!</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p2>
+
+          <h5 class="connect">LET'S CONNECT</h5>
```

Comment:
The design of this element is quite different from the one provided in Figma. You have used the following style for this element.
```
color: blueviolet;
 font-size: 20px;
 font-family: Arial, Helvetica, sans-serif;
```
The actual styling is:
```
font-family: 'Poppins';
font-size: 16px;
color: #7F8CFF;
```